### PR TITLE
Added new data type and programming language

### PR DESCRIPTION
**Pull request by sgosline**

### CODE REVIEW

**Comment by kdaily** [comment]
Please update your base branch to `develop`, @sgosline

**Comment by sgosline** [comment]
Cool, sorry I thought it was on develop...

**Comment by kdaily** [comment]
that's what they all say...

thanks!

File: synapseAnnotations/data/experimentalData.json [hunk 1]
```diff
@@ -300,6 +300,11 @@
         "value": "surveyData",
         "description": "A data set that contains the outcome of a survey.",
         "source": "http://purl.obolibrary.org/obo/OMIABIS_0000060"
+      },
+      {
+        "value": "proteinProteinInteractionNetwork",
```
Comment (kdaily) [inline synapseAnnotations/data/experimentalData.json]: think we decided moving forward to not use camelCase for values? So, can this be `protein-protein interaction network` which is what it is at the source?

Comment (sgosline) [reply synapseAnnotations/data/experimentalData.json]: Because it looks nicer?  *sigh*. will change.

**Comment by kdaily** [comment]
#206

I'm just following orders from...[myself](http://gph.is/1oqSMCz)?

**Pull request merged**